src/System.IO.UnmanagedMemoryStream/tests/UmsWrite.cs

```diff
@@ -111,7 +111,7 @@ public static void CannotWriteWithOverflow()
 
                 if (IntPtr.Size == 4)
```

Comment:
We should remove this whole `if (IntPtr.Size == 4)` path. The other path should work on all platforms now.